src/pipx/commands/upgrade.py

```diff
@@ -137,6 +137,9 @@ def _upgrade_venv(
                 include_dependencies=False,
                 preinstall_packages=None,
                 python_flag_passed=python_flag_passed,
+                # What to deal with this? Should we allow cli option?
+                backend=DEFAULT_BACKEND,
+                installer=DEFAULT_INSTALLER,
```

Comment:
Since there is an option `--install` in upgrade command, perhaps we should allow them to pass `--backend` and `--installer` as well? But this would be quite confusing when people are upgrading packages only instead of installing. How about we add a conditional check to see whether these arguments are used together with `--install` and fall back with a warning if the package exists?

Reply:
Sounds good.